selecta

```diff
@@ -276,6 +266,15 @@ class Search
     end
   end
 
+  def move(direction)
```

Comment:
It seems like you could extract a constant here i.e., `move(UP)` or
`move(:up)`, but I agree.  If we're sticking with numbers, Gary has the
directions correct.

On Mon, Jan 19, 2015 at 9:55 PM, Gary Bernhardt <notifications@github.com>
wrote:

> In selecta
> <https://github.com/garybernhardt/selecta/pull/72#discussion-diff-23200431>
> :
>
> > @@ -276,6 +266,15 @@ class Search
> >      end
> >    end
> >
> > +  def move(direction)
>
> I think that direction should be inverted, so that up is move(-1) and
> down is move(1). That corresponds to the movement through the array.
> AFAIK, there's no standard for whether "up" in a terminal is positive or
> negative, so there's no expectation that will be broken.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/garybernhardt/selecta/pull/72/files#r23200431>.
>



-- 
Thanks,
Taylor Blau

ttaylorr.com